api/src/client/ApiFormationHelpers.ts

```diff
@@ -42,6 +42,15 @@ export const makePostBody = (
     return currentBusiness.profileData.legalStructureId as FormationLegalType;
   };
 
+  const isStartingBusinessAddressPartiallyEmpty = (): boolean => {
+    return (
+      currentBusiness.profileData.businessPersona === "STARTING" &&
+      (formationFormData.addressLine1 === "" ||
+        formationFormData.addressMunicipality === undefined ||
+        formationFormData.addressZipCode === "")
+    );
+  };
```

Comment:
[pebble] The function does have a clear name, but given there are 4 independent checks occurring, I think we can make this a little more explicit. If we set the conditions we're concerned about to variables with clear names it makes the logical operation a little clearer.
```suggestion
  const isStartingBusinessAddressPartiallyEmpty = (): boolean => {
   const isStartingBusiness = currentBusiness.profileData.businessPersona === "STARTING"
   const partiallyCompletedAddress = formationFormData.addressLine1 === "" ||
        formationFormData.addressMunicipality === undefined ||
        formationFormData.addressZipCode === ""
    return isStartingBusiness && partiallyCompletedAddress;
  };
```

Reply:
[boulder] We need to have a test around this logic

Reply:
added tests